blockexplorer/store/config.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2022 Gobalsky Labs Limited
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at https://www.mariadb.com/bsl11.
+//
+// Change Date: 18 months from the later of the date of the first publicly
+// available Distribution of this version of the repository, and 25 June 2022.
+//
+// On the date above, in accordance with the Business Source License, use
+// of this software will be governed by version 3 or later of the GNU General
+// Public License.
+
+package store
+
+import (
+	"code.vegaprotocol.io/vega/libs/config"
+)
+
+var namedLogger = "postgres.store"
+
+type Config struct {
+	Postgres config.PostgresConnection `group:"database" namespace:"postgres"`
+}
+
+func NewDefaultConfig() Config {
+	return Config{
+		Postgres: config.PostgresConnection{
```

Comment:
we'd need a new database every time we restart from a checkpoint right?

Reply:
yeah, that is a good point actually I am not sure what tendermint will do actually when it starts a new chain when there is stuff already in the database from an old one. I suspect just write it alongside what is there, so I probably need to add an 'unsafe-reset-all' command or something - and/or make chainID a mandatory field when querying.